huaweicloud/services/elb/resource_huaweicloud_elb_loadbalancer.go

```diff
@@ -438,6 +460,22 @@ func resourceLoadBalancerV3Update(ctx context.Context, d *schema.ResourceData, m
 			updateOpts.IpV6VipSubnetID = &v6SubnetID
 		}
 
+		if d.HasChange("enable") {
+			enable := d.Get("enable").(bool)
+			updateOpts.AutoScaling = &loadbalancers.AutoScaling{
+				Enable: enable,
+			}
+			if v, ok := d.GetOk("l4_flavor_id"); ok && !enable {
+				updateOpts.L4Flavor = v.(string)
+			}
+			if v, ok := d.GetOk("l7_flavor_id"); ok && !enable {
+				updateOpts.L7Flavor = v.(string)
+			}
+			if v, ok := d.GetOk("min_l7_flavor_id"); ok {
+				updateOpts.AutoScaling.MinL7Flavor = v.(string)
```

Comment:
we can set the value directly as it will be omitted if empty in SDK.

Reply:
done